pages/1.11/monitoring/logging/access-component-logs/index.md

```diff
@@ -29,7 +29,9 @@ Here is the [permission](/1.11/security/ent/perms-reference/) that is required t
     ![New user](/1.11/img/new-user-generic.png)
 
 1.  From the **Permissions** tab, select **ADD PERMISSION**.
-    
+
+    ![Add permission to user](/1.10/img/permission-user.png)
```

Comment:
@joerg84 Added the 1.10 link, should work now vs. removing it entirely.  Leaving it to you to merge. 👍